src/pages/patch/ParametersModal.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Badge from "@leafygreen-ui/badge";
+import { Modal } from "components/Modal";
+import { Parameter } from "types/patch";
```

Comment:
This is being generated and exported from our generated Types

`import {Parameter} from "gql/generated/types"`